script/import_scripts/vbulletin.rb

```diff
@@ -718,22 +722,20 @@ def suspend_users
 
     system_user = Discourse.system_user
 
-    mysql_query("SELECT userid, bandate FROM #{TABLE_PREFIX}userban").each do |b|
-      user = User.find_by_id(b['userid'])
-      if user
-        user.suspended_at = parse_timestamp(user["bandate"])
-        user.suspended_till = 200.years.from_now
-
-        if user.save
-          StaffActionLogger.new(system_user).log_user_suspend(user, "banned during initial import")
-          banned += 1
-        else
-          puts "Failed to suspend user #{user.username}. #{user.errors.try(:full_messages).try(:inspect)}"
-          failed += 1
+    User.find_each do |user|
```

Comment:
Looping through all the users does not seem like a better approach here.

I know that current `suspend_users` method is broken and I have a (single line) fix for this in place which I will commit soon. Can you please omit this change?